src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -350,18 +333,44 @@ function Content(props: {
           onEdit={() => {
             onAccessCodeEdit(accessCode.access_code_id)
           }}
+          onDeleteSuccess={() => {
+            onAccessCodeDeleteSuccess(accessCode.access_code_id)
+          }}
         />
       ))}
     </>
   )
 }
 
+const defaultAccessCodeFilter = (
+  accessCode: AccessCode,
+  searchInputValue: string
+): boolean => {
+  const value = searchInputValue.trim().toLowerCase()
+  if (value === '') return true
+  const name = accessCode.name ?? ''
+  const code = accessCode.code ?? ''
+  return (
+    name.trim().toLowerCase().includes(value) ||
+    code.trim().toLowerCase().includes(value)
+  )
+}
+
+const accessCodeResultToMessage = (
+  result: 'created' | 'updated' | 'deleted'
+): string => {
+  if (result === 'created') return t.accessCodeCreated
+  if (result === 'deleted') return t.accessCodeDeleted
+  return t.accessCodeUpdated
```

Comment:
Just curious, does TypeScript get angry if you do this?

```suggestion
  if (result === 'updated') return t.accessCodeDeleted
```

Reply:
Yes 😅